spa/src/components/account/SignUp/SignUpForm.js

```diff
@@ -60,14 +60,14 @@ function SignUpForm({ onSubmitSignUp, loading }) {
   const disabled = !watchEmail || !watchPassword || loading || !checked;
 
   return (
-    <form onSubmit={disabled ? () => {} : handleSubmit(onSubmit)}>
+    <form onSubmit={disabled ? () => {} : handleSubmit(onSubmitSignUp)}>
```

Comment:
You already disabled the button, no need to have this here
```suggestion
    <form onSubmit={handleSubmit(onSubmitSignUp)}>
```

Reply:
thats because checked is a muicheckbox. letting it be for now.